src/test/scala/org/broadinstitute/dsde/firecloud/dataaccess/MockThurloeDAO.scala

```diff
@@ -106,7 +106,13 @@ class MockThurloeDAO extends ThurloeDAO {
     val newKVsForUser = (userInfo.id -> (mockKeyValues(userInfo.id) ++ keyValues.map { case (key, value) => FireCloudKeyValue(Option(key), Option(value))}))
     mockKeyValues = mockKeyValues + newKVsForUser
     Future.successful(Success(()))
+  }
+
 
+  override def saveKeyValues(forUserId: String, callerToken: WithAccessToken, keyValues: Map[String, String]): Future[Try[Unit]] = {
+    val newKVsForUser = (forUserId -> (mockKeyValues(forUserId) ++ keyValues.map { case (key, value) => FireCloudKeyValue(Option(key), Option(value))}))
```

Comment:
Splitting the line above into multiple lines could make it more self-documenting.